src/Entity/ApiConfigurationInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Synolia\SyliusAkeneoPlugin\Entity;
+
+use Sylius\Component\Resource\Model\ResourceInterface;
+
+/**
+ * @deprecated
+ */
+interface ApiConfigurationInterface extends ResourceInterface
```

Comment:
I don't understand why we add an already deprecated interface?

Reply:
oups, moving it to PR #118

Reply:
moving it to the next PR #118